Samples/iOS-ObjectiveC/iOS-ObjectiveC/ViewController.m

```diff
@@ -99,6 +100,19 @@ - (IBAction)captureException:(id)sender
     [SentrySDK captureException:exception withScope:scope];
 }
 
+- (IBAction)captureTransaction:(id)sender {
+    NSDate* start = [NSDate dateWithTimeIntervalSinceNow:-20];
+    SentryTransactionContext* context = [[SentryTransactionContext alloc] initWithName:@"Other Transaction"];
+    SentryTransaction* fakeTransaction = [[SentryTransaction alloc] initWithTransactionContext:context andHub:SentrySDK.currentHub];
+    
+    //for test purpose, setter will be removed and startTimestamp will be set at initialization
+    fakeTransaction.startTimestamp = start;
+    fakeTransaction.timestamp = [start dateByAddingTimeInterval:2];
+    [fakeTransaction finish]; //use finish instead of setting timestamp
+    
+    [[SentrySDK.currentHub getClient] captureTransaction:fakeTransaction];
```

Comment:
once we get `captureTransaction` to `SentryHub`, we can get rid of this, and on `[fakeTransaction finish]` we'll capture it to Sentry